src/app/music-data/screens/albums-screen.component.tsx

```diff
@@ -1,10 +1,40 @@
-import React from 'react';
-import { View, Text } from 'react-native';
+import React, { useCallback } from 'react';
+import { FlatList } from 'react-native';
+import Icon from 'react-native-vector-icons/dist/Ionicons';
+import { useTheme } from 'styled-components';
+import styled from 'styled-components/native';
+
+import { albums } from '../../../mocks/albums';
```

Comment:
Stop using this. Now you should pass params to the screen in initialScreenOptions prop of Stack.Screen component in routing.